### ISSUE TITLE
Debug version of vtkIOImage-7.1.dll using release version of zlib and libpng

### ISSUE THREAD

**Issue by seanlis** [Microsoft/vcpkg#1326]
**Environment**
Windows 10 Pro 15063.413
vtk:x64-windows             7.1.0
**Steps to Reproduce**
`dumpbin /imports D:\src\vcpkg\installed\x64-windows\debug\bin\vtkIOImage-7.1.dll`
**Expected Results**
```
...
libpng16d.dll
...
zlibd1.dll
...
```
**Actual Results**
```
...
libpng16.dll
...
zlib1.dll
...
```

**Comment by Fleischner** [comment]
I use this issue to promote two additional checks that I run currently by hand:

After build run the following two searches:
A) Search in all release log files (All files with name matches *rel* in folder vcpkg\buildtrees\portname) for Tripletname+"debug\lib" => No matches allowed (otherwise release links to a debug lib)
B) Search in all debug log files (All files with name matches *dbg* in folder vcpkg\buildtrees\portname) for Tripletname+"\lib" => No matches allowed (otherwise debug links to a release lib)

In the unlikely case that I find some time I will add python scripts for all proposed consistency checks.

**Comment by albertziegenhagel** [comment]
This should be fixed by the PR #1601

**Comment by JackBoosY** [comment]
Please open a new issue if this is still a problem for you.
Thx.
